frontend/src/reducers/viewHandler.js

```diff
@@ -404,25 +404,29 @@ export default function viewHandler(state = initialState, action) {
       const type = getViewType(isModal);
       const view = getLocalView(state, id, isModal);
 
-      return {
-        ...state,
-        [`${type}`]: {
-          ...state[`${type}`],
-          [`${id}`]: {
-            ...view,
-            isShowIncluded: !!showIncludedView,
-            hasShowIncluded: !!showIncludedView,
+      if (view.windowId) {
+        return {
+          ...state,
+          [`${type}`]: {
+            ...state[`${type}`],
+            [`${id}`]: {
+              ...view,
+              isShowIncluded: !!showIncludedView,
+              hasShowIncluded: !!showIncludedView,
```

Comment:
out of curiosity: why do we need to use double negation `!!canDoSomething`?

Reply:
@TheBestPessimist  defensively, see https://stackoverflow.com/questions/10467475/double-negation-in-javascript-what-is-the-purpose